bitnami/kafka/templates/statefulset.yaml

```
@@ -59,6 +59,9 @@ spec:
       {{- if .Values.hostAliases }}
       hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.hostAliases "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.hostNetwork }}
+      hostNetwork: {{ .Values.hostNetwork }}
+      {{- end }}
```

You can simplify this since the parameter is a boolean
```suggestion
      hostNetwork: {{ .Values.postgresql.hostNetwork }}
```